devfiles/che-theia-all.devfile.yaml

```diff
@@ -0,0 +1,187 @@
+# A devfile to setup che-theia projects.
+# For developers who work on Theia core or che-theia core extensions and plugins.
+
+apiVersion: 1.0.0
+metadata:
+  name: che-theia-all
+
+projects:
+
+  - name: che-theia
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che-theia.git'
+
+  - name: theia
+    source:
+      type: git
+      location: 'https://github.com/theia-ide/theia.git'
+
+components:
+
+  - alias: git
+    type: dockerimage
+    image: sunix/git-devtools
```

Comment:
hello, images should be hosted on che(something) namespace with Dockerfile available as well for enhancements

BTW git is included in theia already no ? so why do we need an extra container ?

Reply:
The git provided in the theia container is just good enough to invoke git from Theia. But once you want to use advanced features from the command line, it is very poor.
We are in the middle of a lot of changes in Theia and git stuff but in the mean time, I can be productive only with a good git cli experience.
I am making a lots of changes at the moment on that container and would like to propose that image into a eclipse/something  a bit later if it works for you.